grafana-plugin/src/pages/index.tsx

```diff
@@ -139,7 +138,6 @@ export const pages: { [id: string]: PageDefinition } = [
     icon: 'cloud',
     id: 'cloud',
     text: 'Cloud',
-    role: 'Admin',
```

Comment:
`role` is no longer used, this has been superseded by `action`. This should be safe to remove as the `PageDefinition` type does not contain a `role` attribute so any access to it would throw a type error during build time.